lib/bandiera/gui/views/_feature_form.erb

```diff
@@ -1,7 +1,7 @@
 <%
   @feature ||= Bandiera::Feature.new(
     name:         '',
-    group:        nil,
+    group:        defined?(group).nil? ? nil : group,
```

Comment:
if partial is called with group (happens from group page), then use it for the new feature, otherwise behave as before.
maybe you can suggest a cleaner syntax to get this working? it seems a little overprotective to me but didn't have time to come up with something nicer

Reply:
I'd maybe remove the inverse logic here:

```
defined?(group) ? group : nil
```

as `defined?` will return `true` if `group` is defined - no need to check for `nil`

Reply:
if i remember correctly `defined?` returns nil for undefined vars and their type (as string?) if defined.
but even so i think removing `.nil?` check might work out ok, will try it out :+1:

Reply:
yup works, truthy and falsy for the win, pushing this change now